tv_apps/smart-home/js/card_picker.js

```diff
@@ -86,10 +87,10 @@
 
       this._cardManager.getCardList()
         .then(cardList => {
-          this._refreshCardButtons(folderList, cardList);
+          this._refreshCardElements(folderList, cardList);
           this._spatialNavigator.setCollection(
-                            this.appButtons.concat(this.navigableElements));
-          this._spatialNavigator.focus(this.appButtons[0]);
+                      this.navigableElements.concat(Array.from(this.allItems)));
+          this._spatialNavigator.focus(this.allItems[0]);
```

Comment:
I think `allItems[0]` is not the first item of collection here (but in old version it is). But I can change it back to the old version and remove the parameter if you think that's better.